00-repos-centos7.ks

```diff
@@ -1,5 +1,10 @@
 repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
 repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
 repo --name=epel7 --mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
-repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/nightly/el7/$basearch/
-repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/nightly/el7/$basearch/
+repo --name=centos-sclo --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
```

Comment:
```suggestion
repo --name=centos-sclo-rh --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
```

I agree with @ekohl, let's add `-rh` suffix to the name

Reply:
There repositories do not make it to the image, it's used only during the livecd creation. But I can change it.